phpcs.xml.dist

```diff
@@ -77,6 +77,7 @@
 	<!-- Exclude PHPUnit tests from file and class name sniffs (for Core parity). -->
 	<rule ref="WordPress.Files.FileName.NotHyphenatedLowercase">
 		<exclude-pattern>/phpunit/*</exclude-pattern>
+		<exclude-pattern>*.asset.php</exclude-pattern>
```

Comment:
It will work, but I'd prefer if the `exclude-pattern` were more specific to not exclude asset files in other directories:

```suggestion
<exclude-pattern>/packages/e2e-tests/.+\.asset\.php$</exclude-pattern>
```

If the intention is to ignore asset files in all directories, I'd suggest using this pattern:
```suggestion
<exclude-pattern>*\.asset\.php$</exclude-pattern>
```

Reply:
I think it makes sense to exclude all `*.asset.php` files from this rule. It's a common name with a specific form and it's unlikely that we'll name other files in this way by accident.